.github/workflows/test.yml

```diff
@@ -1,6 +1,6 @@
 name: Test
 
-on: [push, pull_request]
+on: [pull_request]
```

Comment:
Is this intended? if so maybe split it in another commit

Reply:
yes, I noticed it cause to start the CI 2 times in most case, will split commits